src/main.cpp

```diff
@@ -420,11 +432,11 @@ void connectionWatchdogCallback()
     ESP.restart();
 }
 
-void configureMQTTDiscovery(String device_name, String component, String mqtt_command, String mqtt_status)
+void configureMQTTDiscovery(String prefix, String device_name, String component, String mqtt_command, String mqtt_status)
```

Comment:
Discovery prefix should be the same for all components

Reply:
Actually I have the use case where I want different discovery prefixes. I installed two double ewelinks. The first one control the corridor light and a room light. The second one the bathroom light and `the same corridor light`. I configured the same mqtt topic, but I didn't enter discovery for that, because I don't want two "corridor" devices in home assistant, for me is the same.